rclrs/src/qos.rs

```diff
@@ -137,3 +177,45 @@ impl From<QoSDurabilityPolicy> for rmw_qos_durability_policy_t {
         }
     }
 }
+
+pub const QOS_PROFILE_SENSOR_DATA: QoSProfile = QoSProfile {
+    history: QoSHistoryPolicy::KeepLast { depth: 5 },
+    reliability: QoSReliabilityPolicy::BestEffort,
+    durability: QoSDurabilityPolicy::Volatile,
+    avoid_ros_namespace_conventions: false,
+};
+
+pub const QOS_PROFILE_PARAMETERS: QoSProfile = QoSProfile {
+    history: QoSHistoryPolicy::KeepLast { depth: 1000 },
+    reliability: QoSReliabilityPolicy::Reliable,
+    durability: QoSDurabilityPolicy::Volatile,
+    avoid_ros_namespace_conventions: false,
+};
+
+pub const QOS_PROFILE_DEFAULT: QoSProfile = QoSProfile {
+    history: QoSHistoryPolicy::KeepLast { depth: 10 },
+    reliability: QoSReliabilityPolicy::Reliable,
+    durability: QoSDurabilityPolicy::Volatile,
+    avoid_ros_namespace_conventions: false,
+};
+
+pub const QOS_PROFILE_SERVICES_DEFAULT: QoSProfile = QoSProfile {
+    history: QoSHistoryPolicy::KeepLast { depth: 10 },
+    reliability: QoSReliabilityPolicy::Reliable,
+    durability: QoSDurabilityPolicy::Volatile,
+    avoid_ros_namespace_conventions: false,
+};
+
+pub const QOS_PROFILE_PARAMETER_EVENTS: QoSProfile = QoSProfile {
+    history: QoSHistoryPolicy::KeepAll,
+    reliability: QoSReliabilityPolicy::Reliable,
+    durability: QoSDurabilityPolicy::Volatile,
+    avoid_ros_namespace_conventions: false,
+};
+
+pub const QOS_PROFILE_SYSTEM_DEFAULT: QoSProfile = QoSProfile {
+    history: QoSHistoryPolicy::SystemDefault { depth: 0 },
```

Comment:
I assume `depth: 0` means the depth is unspecified (decided by the system defaults), and not that there is zero depth in the message buffer. It might be good to leave a comment about that so users don't get alarmed or confused.

Reply:
I don't know if that's the case, so I don't want to promise things I'm not sure about myself. But this was 0 already before this change and even in rclcpp/rmw (in `rmw_qos_profile_system_default`).